src/Monitors/Models.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Collection\Monitors;
+
+use Collection\Monitors\Monitors;
+use PDO;
+
+class Models
```

Comment:
You should rename this class to be more specific, MonitorModel would be the best practice name

Reply:
Renamed class and filename